### PR TITLE
119 search by collections

### PR DESCRIPTION
**Pull request by AlZorba**
**Issue** :  `https://github.com/archaiodata/thacer/issues/119`

**Description** :    
This PR add property 'collection' to pottery objects according to the source file and make it searchable

### CODE REVIEW

**Comment by AlZorba** [comment]
@francoiscrouzet @chtzv I guess I will stay in this state and make a release. What do you think?

File: vue-thacer/src/assets/js/thacer-map-setup-search.js [hunk 1]
```diff
@@ -34,9 +34,9 @@ export function setupSearchCeramByText(markerClusterGroupCeram, map) {
             let label = obj.properties.Pi ? 'Π' + obj.properties.Pi : obj.properties.ID
 
             document.getElementById('nonloc').innerHTML += [
-              '<a class="unlocalised-tag px-1 m-0 border border-white" href="ceram?ID=' +
+              '<a class="unlocalised-tag px-1 m-0 border border-white" href="#/ceram?ID=' +
                 obj.properties.ID +
-                '&ANA=8888880&INV=88880">' +
+                '">' +
```
Comment (francoiscrouzet) [inline vue-thacer/src/assets/js/thacer-map-setup-search.js:39]: aaah now I see

File: vue-thacer/src/assets/js/thacer-map-setup-search.js [hunk 2]
```diff
@@ -28,7 +28,7 @@ export function setupSearchCeramByText(markerClusterGroupCeram, map) {
 
           // Adding current ceramObject only if unlocalised and passing input search string :
           if (
-            obj.properties.x != '' &&
+            obj.geometry.coordinates[0] === 0 &&
```
Comment (francoiscrouzet) [inline vue-thacer/src/assets/js/thacer-map-setup-search.js:31]: coordinates (and geometry) properties always present ?
(sorry, I write that in every single PR haha)

File: API/CSV/ETHVII.txt [hunk 3]
```diff
@@ -1,873 +1,873 @@
...
-2,1108,"D 223","FRAGMENTS (2) DE BOL A OISEAU.
-Larg. max. 2,2 cm ; haut. max. 2,3 cm. Argile chamois clair. Intérieur, vernis noir brunâtre. Extérieur : oiseau quadrillé à droite.","Lilly Ghali-Kahil, La céramique grecque (fouilles 1911-1956), Paris, De Boccard, Etudes thasiennes 7, 1960, p. 18, no 2 pl. I, 2",,,"fine","rhodienne","bol à oiseau","bol","archaïque",,
-3,,"55-150","FRAGMENT DE BOL A OISEAU.
+"ID","Pi","Inventaire","Famille","Catégorie","Forme","Type","Période","Texte","Bibliographie","secteur","secteur_ID","x","y"
```
Comment (francoiscrouzet) [inline API/CSV/ETHVII.txt:1]: All done by chatGPT ? :)

**Review by francoiscrouzet — APPROVED** [review]
Super !

File: vue-thacer/src/assets/js/thacer-map-setup-search.js (hunk 1 above)
Comment (francoiscrouzet) [reply vue-thacer/src/assets/js/thacer-map-setup-search.js:39]: Well, I would still like to know what this is all about :D

**Comment by francoiscrouzet** [comment]
> @francoiscrouzet @chtzv I guess I will stay in this state and make a release. What do you think?

We were unhappy with our deployment procedure last time. You may want that we do it together to improve it and make it works well

File: vue-thacer/src/assets/js/thacer-map-setup-search.js (hunk 2 above)
Comment (AlZorba) [reply vue-thacer/src/assets/js/thacer-map-setup-search.js:31]: yes, because it is geojson and mandatory

File: API/CSV/ETHVII.txt (hunk 3 above)
Comment (AlZorba) [reply API/CSV/ETHVII.txt:1]: didn't get it !

File: vue-thacer/src/assets/js/thacer-map-setup-search.js (hunk 1 above)
Comment (AlZorba) [reply vue-thacer/src/assets/js/thacer-map-setup-search.js:39]: Trying to find the best way to explain, I think it is just about having to mention the ID in the routes and not ID + ANA + INV which are IDs like use for photo.

**Pull request merged**